test/e2e/containers_conf_test.go

```diff
@@ -346,10 +346,10 @@ var _ = Describe("Verify podman containers.conf usage", func() {
 		Expect(session.OutputToString()).To(ContainSubstring("HST"))
 
 		// verify flag still overrides
-		session = podmanTest.Podman([]string{"run", "--tz", "EST", ALPINE, "date", "+'%H %Z'"})
+		session = podmanTest.Podman([]string{"run", "--tz", "GMT", ALPINE, "date", "+'%H %Z'"})
 		session.WaitWithDefaultTimeout()
 		Expect(session).Should(ExitCleanly())
-		Expect(session.OutputToString()).To(ContainSubstring("EST"))
+		Expect(session.OutputToString()).To(ContainSubstring("GMT"))
```

Comment:
This is not quite useless, but close enough that a future maintainer might think it is: since the default is UTC, it isn't immediately clear that this actually tests the tz code.

Could you try `America/Phoenix` and search for `MST`? Possibly with a comment indicating "Arizona does not observe DST so this command is safe at all times of the year"?

Reply:
done

Reply:
UTC vs GMT is why I said "*not quite* useless"; the thing is, a normal human will know that podman defaults to 0000 but not necessarily know or be certain whether it's gonna say UTC or GMT, and may have to take a few minutes to confirm. This way, it's immediately obvious.

Thank you!